pkg_building.Rmd

```diff
@@ -341,6 +348,10 @@ Once your package is on CRAN, it will be [regularly checked on different platfor
 
 * the [CRAN checks API](https://github.com/ropenscilabs/cchecksapi) maintained by rOpenSci, that provides [badges](https://github.com/ropenscilabs/cchecksapi#badges). You can add one of them to the README of your package via [`rodev::use_cchecks_badge()`](https://docs.ropensci.org/rodev/reference/use_cchecks_badge.html).
 
+## Bioconductor gotchas
```

Comment:
I also think that if the package will be hosted on bioconductor or even depends on Bioconductor packages, this should be flagged in the submission / special checks and instructions potentially included in the review checklist. It may well save reviewers (and editors!) a lot of time if they are aware upfront and perhaps encourage authors to make sure they have addressed any difficulties this may create.

Reply:
Indeed. Sorry for all the trouble!! An many thanks for identifying those problems!

Reply:
good thinking! we'd just need to add a relevant line in https://github.com/ropensci/software-review/blob/master/.github/ISSUE_TEMPLATE/A-submit-software-for-review.md#publication-options

Reply:
No problem @llrs! And thanks for commenting and helping us learn!

Reply:
Yes perhaps a check somewhere to indicate whether a package depends on Bioconductor packages too as being aware of this can speed up troubleshooting any potential installation problems and also focuses the mind of the author too.

Reply:
> Yes perhaps a check somewhere to indicate whether a package depends on Bioconductor packages too as being aware of this can speed up troubleshooting any potential installation problems and also focuses the mind of the author too.

Do you mean it's not enough to document it in the packaging guide, where else should we flag that?

Reply:
So I've had complications in two (to a certain extent related) case:
1) when a package will be hosted on bioconductor.
2) when a package depends on bioconductor packages.

When a package will be hosted on bioconductor packages, I reckon additional requirements come into play like:
- testing on `r: bioc-release`
- installing the package using `BiocManager::install()` or even `remotes::install_bioc()` to install the development version of the package

When a package simply depends on bioconductor packages, there might be other complications, like either checking the `biocViews:` filed exists / manual installation of dependencies / whatever we figure out is the correct way!

I feel both situations are likely to save time during editor's checks / review if they are flagged in the submission template.